linera-storage-service/src/server.rs

```diff
@@ -477,6 +498,15 @@ impl StoreProcessor for ServiceStoreServer {
 
 #[tokio::main]
 async fn main() {
+    let env_filter = tracing_subscriber::EnvFilter::builder()
+        .with_default_directive(tracing_subscriber::filter::LevelFilter::INFO.into())
+        .from_env_lossy();
+    tracing_subscriber::fmt()
+        .with_span_events(FmtSpan::NEW)
```

Comment:
I think it would be really nice to mimic [what `test-log` does here](https://github.com/d-e-s-o/test-log/blob/756d49ee2b96cc93f332549e5173017bebb6be58/macros/src/lib.rs#L202-L224) and support runtime configuration by the `RUST_LOG_SPAN_EVENTS` environment variable.

Reply:
Ok, I implemented that.